app/uk/gov/hmrc/gform/sharedmodel/formtemplate/BooleanExpr.scala

```diff
@@ -54,6 +54,33 @@ object BooleanExpr {
     Format(reads, writes)
   }
 
+  implicit val leafExprs: LeafExpr[BooleanExpr] = new LeafExpr[BooleanExpr] {
+    def exprs(path: TemplatePath, be: BooleanExpr): List[ExprWithPath] = {
+
+      def WithPath(xs: Expr*): List[ExprWithPath] = xs.toList.map(x => ExprWithPath(path, x))
```

Comment:
defs are usually camel case, is this some sort of covention?

Reply:
Yes, the intention was for the calls of `WithPath` to look like calls of constructor